server/entities/dto.go

```diff
@@ -6,6 +6,7 @@ type X509CertificateRequest struct {
 
 type X509CertificateResponse struct {
```

Comment:
Easily extensible struct for the future. I suspect we'll likely need more info from Auth server in the future. Not now, but I imagine this struct could be expanded / called something generic and then we can eventually stuff it with other details.